assets/js/common.js

```diff
@@ -771,8 +771,8 @@ var hivepress = {
 
 		forms.each(function () {
 			var form = $(this),
-				captcha = form.find('.g-recaptcha'),
-				captchaId = $('.g-recaptcha').index(captcha.get(0)),
+				captcha = form.find('.recaptcha'),
```

Comment:
Let me know why changing the class name was needed

Reply:
Because captcha is automatically generated for the old class and we don't need this for v3 of reCaptcha